src/main/java/mate/project/dto/PlaceOrderDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.project.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class PlaceOrderDto {
```

Comment:
The `@NotNull` annotation is used to ensure that the shippingAddress field is not null. However, it does not check for empty strings. Consider using `@NotBlank` from jakarta.validation.constraints if you want to ensure the field is not null and not empty.